### DISCUSSION TITLE
fuck you

### DISCUSSION THREAD

**Discussion by Nrmot** [Nrmot-GithubRepo/Project-CBT#32] [Project CBT: Latest Dev Releases]
fuck you

changes: added epic siege mod, opencomputers, and more

added logic gates made of 4sa3nq

added totally not sus recipes

thats all

<hr /><em>This discussion was created from the release <a href='https://github.com/Nrmot-GithubRepo/Project-CBT/releases/tag/Alpha-0.0.3-snapshot1'>fuck you</a>.</em>

**Comment by Higuys153** [comment]
ah so this is why i got an email titled fuck you

**Reply by Nrmot** [reply]
most sane release title :trollface:
